hdfs-scheduler/src/main/java/org/apache/mesos/hdfs/state/LiveState.java

```diff
@@ -151,4 +151,60 @@ public boolean apply(String taskId) {
       }
     }).size();
   }
+
+  private class NameNodeState {
+    private String name;
+    private TaskStatus status;
+    private Boolean initialized;
+
+    public NameNodeState(String name) {
+      this.name = name;
+      clear();
```

Comment:
Why would we need to clear within the constructor. This is when we are creating the object so everything should be clear already.